ui-components/src/locales/general.json

```diff
@@ -1325,21 +1325,19 @@
       "header" : "Let's find your next home",
       "sections": {
         "welcome" : "Welcome",
-        "bedrooms" : "Bedrooms",
+        "persons" : "Persons",
```

Comment:
@tomgebauer to weigh in on this label for this page of the questionnaire - "Persons"? "Household"? "Household size"?

Reply:
Household size sounds like the most straightforward.

Reply:
Updated to Household Size, per our chat.
![image](https://user-images.githubusercontent.com/82653098/131711301-99b54aba-3d89-4ce2-9aad-ac3dc5a98104.png)